andela/devrepublic-bn-backend
# #170947542 setup docker

**What does this PR do?**
Setup Docker

**Description of Task to be completed?**
* add Dockerfile
* add dockerignore file to ignore .dockerignore

**Any background context you want to provide?**
* N/A

**How should this be manually tested?**
* Install docker in your local machine
* run `docker build -t barefoot-nomad .`
* run `docker run -it -p 9000:3000 barefoot-nomad`
* go to your browser and check localhost:9000

**What are the relevant pivotal tracker stories?**
[#170947542](https://www.pivotaltracker.com/story/show/170947542)

**Screenshots (if appropriate)**
* N/A
